tests/rptest/tests/partition_reassignments_test.py

```diff
@@ -437,6 +438,30 @@ def test_reassignments_cancel(self):
         check_verify_reassign_partitions(output, reassignments_json,
                                          self.logger)
 
+    @cluster(num_nodes=4)
+    def test_disable_alter_reassignment_api(self):
+        # works
+        kcl = KCL(self.redpanda)
+        kcl.alter_partition_reassignments({})
+
+        # disable api
+        rpk = RpkTool(self.redpanda)
+        rpk.cluster_config_set("kafka_enable_partition_reassignment",
```

Comment:
RedpandaService.set_cluster_config will set the config and wait for it to propagate, so that you don't have to wait_until anything in the test.

Reply:
Thanks. I thought this existed, but I was looking for it in admin.py